internal/buildscripts/packaging/choco/splunk-otel-collector/tools/chocolateyinstall.ps1

```diff
@@ -21,6 +23,22 @@ if ($MODE -and ($MODE -ne "agent") -and ($MODE -ne "gateway")) {
     throw "Invalid value of MODE option is specified. Collector service can only run in agent or gateway mode."
 }
 
+if ($pp['WITH_FLUENTD']) {
+    if (($pp['WITH_FLUENTD'] -eq "true") -or ($pp['WITH_FLUENTD'] -eq "false")){
+        try {
+            $WITH_FLUENTD = [System.Convert]::ToBoolean($pp['WITH_FLUENTD']) 
+        } catch [FormatException] {
+            $WITH_FLUENTD = $FALSE
+        }
+    }
+    else {
+        throw "Invalid value of WITH_FLUENTD option is specified. Possible values are true and false."
+    }
+}
+if ($WITH_FLUENTD) {
+    . $toolsDir\fluentd.ps1

```

Comment:
The fluentd installation should be moved to after the splunk-otel-collector installation since the fluentd configuration requires the config files provided by the splunk-otel-collector.